### ISSUE TITLE
dnote login to custom endpoint from the cli

### ISSUE THREAD

**Issue by navado** [dnote/dnote#425]
* **Please describe the feature you would like to have.**
Ability to login to privatly hosted endpoint purely from the CLI.
```
dnote login -e https://my.self.hosted.endpoint/api
```
The rest of behavior may be as today.

**Comment by sungwoncho** [comment]
@navado Good idea. We can define a flag in [here](https://github.com/dnote/dnote/blob/master/pkg/cli/cmd/login/login.go) to override the configuration that comes from `.dnoterc`. Just curious, what would be a primary use case for such a feature?

**Comment by navado** [comment]
I thought about creating the configuration without using editor and not actually care where it's located.
